testing/go/functions_test.go

```diff
@@ -434,5 +434,41 @@ func TestSystemInformationFunctions(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "col_description",
+			Assertions: []ScriptTestAssertion{
+				{
+					Query: `SELECT col_description(100, 1);`,
```

Comment:
For these, I think it would be better to also create a case where it should return something and make it a skipped test. Like `SELECT obj_description('sinh'::regproc, 'pg_proc');` returns `”hyperbolic sine”`.